lib/modules/onc_program/bulk_data_group_validation_sequence.rb

```diff
@@ -282,7 +282,7 @@ def streamed_ndjson_get(url, headers)
         end
 
         if line_count > MAX_RECENT_LINE_SIZE
-          response_for_log[:body] = "NOTE: RESPONSE TRUNCATED\nINFERNO ONLY DISPLAYS MOST RECENT #{MAX_RECENT_LINE_SIZE} LINES\n\n#{response_for_log[:body]}"
+          response_for_log[:body] = "NOTE: RESPONSE TRUNCATED\nINFERNO ONLY DISPLAYS FHIRST #{MAX_RECENT_LINE_SIZE} LINES\n\n#{response_for_log[:body]}"
```

Comment:
FHIRst 😃 

Did you change the logger behavior so it saves the first X?  I think I had it at 'most recent' before, but i agree that given how we are referencing line numbers for errors this previous behavior made it so you couldn't figure out what line the errors referred to.

Reply:
Sometime, I cannot even spell the word "fire" correctly. Yes I did change the order of logs kept. Typo fixed